pkg/console/starter/starter.go

```diff
@@ -161,17 +163,30 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	versionRecorder.SetVersion("operator", os.Getenv("RELEASE_VERSION"))
 
 	clusterOperatorStatus := status.NewClusterOperatorStatusController(
-		"console",
+		// name
+		api.OpenShiftConsoleName,
+		// related objects
 		[]configv1.ObjectReference{
-			{Group: "operator.openshift.io", Resource: "consoles", Name: api.ConfigResourceName},
-			{Group: "config.openshift.io", Resource: "consoles", Name: api.ConfigResourceName},
-			{Group: "oauth.openshift.io", Resource: "oauthclients", Name: api.OAuthClientName},
-			{Resource: "namespaces", Name: api.OpenShiftConsoleOperatorNamespace},
-			{Resource: "namespaces", Name: api.OpenShiftConsoleNamespace},
+			// top level configs
+			{Group: configv1.GroupName, Resource: "consoles", Name: api.ConfigResourceName},
+			{Group: configv1.GroupName, Resource: "infrastructures", Name: api.ConfigResourceName},
+			// operator configs
+			{Group: operatorv1.GroupName, Resource: "consoles", Name: api.ConfigResourceName},
+			// resources
+			{Group: oauthv1.GroupName, Resource: "oauthclients", Name: api.OAuthClientName},
+			{Group: corev1.GroupName, Resource: "namespaces", Name: api.OpenShiftConsoleOperatorNamespace},
+			{Group: corev1.GroupName, Resource: "namespaces", Name: api.OpenShiftConsoleNamespace},
 		},
+
+		// cluster operator client
 		configClient.ConfigV1(),
+		// cluster operator informer
+		configInformers.Config().V1().ClusterOperators(),
```

Comment:
not found, followed by already exists for an extended period of time suggests an informer/lister isn't getting started.